invenio_app_ils/config.py

```diff
@@ -252,7 +272,7 @@ def _(x):
 # Database
 ###############################################################################
 #: Database URI including user and password
-SQLALCHEMY_DATABASE_URI = "postgresql+psycopg2://test:psw@localhost/ils"
+SQLALCHEMY_DATABASE_URI = "postgresql+psycopg2://invenio:invenio@localhost/ils"
```

Comment:
to be reverted?

Reply:
I cannot run it with docker-services-cli if I don't configure it in this way, I had the same problem with cds-ils

Reply:
let's see this together, I am worried that after merging this everyone else will be impacted.